101-data-share-share/azuredeploy.json

```diff
@@ -0,0 +1,39 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "account_name": {
+            "type": "String"
+        },
+        "location": {
+            "type": "string"
+        },
+        "share_name": {
+            "type": "String"
+        }
+    },
+    "variables": {},
+    "resources": [
+        {
+            "type": "Microsoft.DataShare/accounts",
+            "apiVersion": "2019-11-01",
+            "name": "[parameters('account_name')]",
+            "location": "[parameters('location')]",
+            "identity": {
+                "type": "SystemAssigned"
```

Comment:
how is this used?

Reply:
> how is this used?

user has to enter their account and share name to create a share under that account.

Reply:
not the resource - but the MSI the resource is creating - how is that used?

Reply:
How are permissions granted to the storage account?

Reply:
@bmoore-msft Data Share resource's managed identity needs to be granted access to Azure data store. Azure Data Share service then uses this managed identity to read and write data for snapshot-based sharing, and to establish symbolic link for in-place sharing. For more details you can refer to this link: https://docs.microsoft.com/en-us/azure/data-share/concepts-roles-permissions#roles-and-requirements

Reply:
Got it - let's add this information to the readme so other users will know how it works as well...

Reply:
added.